### PR TITLE
Add 5 more ICEs

### PR DESCRIPTION
**Pull request by Julian-Wollersberger**
Added 
* rust-lang/rust#67945 
* rust-lang/rust#67981 
* rust-lang/rust#68013 
* rust-lang/rust#44861 
* rust-lang/rust#68025

### CODE REVIEW

File: ices/44861.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# https://github.com/rust-lang/rust/issues/44861
+# Playground: https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=bdd59503a0dc2d33c60a949fc7c57633
+# You need to select `build` instead of `run`.
+rustc --crate-type lib - << END
```
Comment (JohnTitor) [inline ices/44861.sh]: This can be `.rs` file by using `crate_type` attribute, I think?

**Comment by JohnTitor** [comment]
And you mention to rust-lang/rust#66930 but this doesn't touch and the test case is already added, it's a simple mistake, I guess.

File: ices/44861.sh (hunk 1 above)
Comment (Julian-Wollersberger) [reply ices/44861.sh]: Yes, thanks for the suggestion!

**Comment by JohnTitor** [comment]
Thanks!

**Pull request merged**